sorc/grid_tools.fd/regional_esg_grid.fd/pmat5.f90

```diff
@@ -5,30 +5,27 @@
 !!
 !! DEPENDENCIES
 !! Modules:  pkind, pietc, pmat4
-!!
+!> Constants for orientation and stretching of map
```

Comment:
You need a blank line here to break this into two doxygen blocks, one for the file, one for the module.